quotes/ContentView.swift

```diff
@@ -8,7 +8,7 @@ import SwiftUI
 
 struct ContentView: View {
     @State var quote: Quote?
-    var body : some View {
+    var body: some View {
         VStack {
             if let q = quote {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/swift-linting-formatting-workshop/blob/6bd6a61968c868f71f3fef73fa1bffcb3d3a399c/quotes/ContentView.swift#L13"></span>Variable name should be between 3 and 40 characters long: 'q'<br />
<code>identifier_name</code> <code>ContentView.swift:13</code></td>
    </tr>
  </tbody>
</table>